styx-scheduler-service/src/test/java/com/spotify/styx/state/handlers/FlyteRunnerHandlerTest.java

```diff
@@ -165,7 +154,43 @@ public void shouldNotThrowExceptionIfEvenRouterIsClosed(State state) throws Exce
         .build());
 
     flyteRunnerHandler.transitionInto(runState, eventRouter);
+  }
+
+  @Test
+  public void shouldPollInRunning() throws FlyteRunner.PollingException {
+    RunState runState = RunState.create(WORKFLOW_INSTANCE, State.RUNNING, StateData.newBuilder()
+        .executionId(EXECUTION_ID)
+        .executionDescription(FLYTE_EXECUTION_DESCRIPTION)
+        .build());
+
+    flyteRunnerHandler.transitionInto(runState, eventRouter);
+
+    verify(flyteRunner).poll(getFlyteExecutionId(FLYTE_EXECUTION_DESCRIPTION,
+        reverse.apply(EXECUTION_ID)), runState);
```

Comment:
Why do we need to use reversed `EXECUTION_ID`?

Reply:
The styx execution id `styx-run-xxxxxx` can not be used as is in Flyte, since Flyte has a limitation of (i think) 20 characters for the specific id.
So we generate it first and then transform it before passing it along in Flyte.

Reply:
Glad that you asked, it shows that it is not clear what are we doing here.
The issue is that we cannot use straight styx-run-ids as flyte-exec-names because flyte name are shorter. So a transformation function is needed. With the purpose of making the testing easier we pass a `Function<String, String>` as parameter to be able to use a simpler one (string reverse) for testing instead that the real one that it is more complicated.

To improve readability, we can create a constant:
```java
  private static final String FLYTE_EXEC_NAME = reverse.apply(EXECUTION_ID);
```
and then use the constant directly in the verify.

Side note: at the beginning we didn't use reverse but `Function.identity()` but this hid a bug that we were passing the styx-run-id instead fo the flyte exec name. The idea of using reverse was to do a simple transformation that it is easy to check.

Reply:
Does it makes sense @tnsetting?. By the way it took me like a second to recognize that this was your github account nian

Reply:
@narape Yes. It is has been a while. 

That makes sense to me (didn't not realise the execution mapping part) and it sounds to use a constant to improve readability. 👍